docs/UserGuide.adoc

```diff
@@ -641,6 +656,6 @@ for harmful interactions between patient and donor's blood
 
 * duplicate: refers to two persons of the same type which have the same NRIC
 
-* success rate: refers to a number which indicates the degree of compatibility between the patient and donor. This percentage is calculated from the number of tissue matches between the patient and donor.
+* compatibility rate: refers to a number which indicates the degree of compatibility between the patient and donor. This percentage is calculated from the number of tissue matches between the patient and donor.
```

Comment:
~~ refers ...~~ A percentage from 0 to 100% that indicates the degree of compatibility between the patient and the donor.

Reply:
might be redundant to say percentage from 0% to 100% hmmm

Reply:
Noted